### ISSUE TITLE
how to lower loss after training 

### ISSUE THREAD

**Issue by peishanfelicia** [AlexeyAB/darknet#1962]
Hi, I am currently training YOLOv2 for object detection (on only one object) using this command darknet.exe detector train data/obj.data yolo-obj.cfg darknet53.conv.74 . I have two questions in mind. 
Question 1. So, I was wondering what are the methods to reduce the loss of the detector model? I have read about optimizer , backpropgation and loss function of YOLOv2.
Q2) How do I implement backpropagation and reduce my loss for the detector. 

Thank you.
Cheers, felicia
